### PR TITLE
Add contact form section

### PR DESCRIPTION
**Pull request by Stephen-Adom**
The contact form has been made according to the following requirements:
1- The name field accepts up to 30 chars.
2- The email field accepts only emails.
3- The message field accepts up to 500 chars.
4- All fields have placeholder text.
5- All fields have the correct type.
6- A user cannot submit the form if one or more of the fields is empty

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -341,38 +313,69 @@ <h1>About me</h1>
...
+              id="email"
+              placeholder="Email address" required
+            />
+            <textarea
+              name="textarea"
+              id="textarea"
+              cols="5"
+              rows="5"
+              placeholder="Enter text here" maxlength="500" required
+            ></textarea>
```
Comment (iambenkis) [inline index.html:338]: - [ ] I was about to approve your project for the wonderful work you did :clap:, but I noticed that the form should send a message with a successful response. Kindly make sure you are able to receive a message with on your email address , When you click on the `Get in touch` button . :climbing_woman: 

Screenshot from 2023-01-17 17-38-35 (1)

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Stephen-Adom  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zunairkhan811) [reply index.html:338]: image

Hi! I think you overlooked the code. The changes you requested I have already made the changes you asked for.

**Review by huseyinbiyik — APPROVED** [review]
Hi team! Brilliant job!

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Collaborative work! 👌👌
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Optional suggestions
- You had really good and descriptive commit messages but let's capitalized the first word on the message. It is the best practice for readability.
Screenshot 2023-01-17 at 7 24 19 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._∂

**Pull request merged**